pom.xml

```diff
@@ -289,7 +289,7 @@
             <dependency>
                 <groupId>org.jdbi</groupId>
                 <artifactId>jdbi3-bom</artifactId>
-                <version>3.45.3</version>
+                <version>3.45.4</version>
```

Comment:
> Update flyway to 3.45.4

The commit title seems wrong.

Reply:
Yeah. Thanks for catching up. I'll amend before merging